packages/cryptoassets/src/currencies.ts

```diff
@@ -78,6 +78,25 @@ const ethereumUnits = (name, code) => [
 ];
 
 const cryptocurrenciesById: Record<string, CryptoCurrency> = {
+  near: {
+    type: "CryptoCurrency",
+    id: "near",
+    coinType: 0, //TODO: what should I put here?
+    name: "NEAR", //TODO: is it the name of the company, or name of the cryptocurrency?
+    managerAppName: "NEAR",
+    ticker: "N",
```

Comment:
mmh I think the ticker used in markets are `NEAR`, no? https://www.coingecko.com/fr/pi%C3%A8ces/near#markets